server/master/src/main/java/org/apache/accumulo/master/tableOps/tableImport/ImportTable.java

```diff
@@ -19,13 +19,14 @@
 package org.apache.accumulo.master.tableOps.tableImport;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.function.Predicate.not;
```

Comment:
@ctubbsii @keith-turner - `Predicate.not` was added in Java 11 and I expected the build to fail when I used it, but it didn't. What runtimes are we supporting for 2.1.x?

Reply:
2.1 is targeting Java 11.